loom/script/compiler/lsParser.cpp

```diff
@@ -2572,6 +2572,9 @@ Statement *Parser::parseEnumStatement()
     cls->metaTags = curTags;
     curTags.clear();
 
+    cls->docString = currentMultilineComment;
+    currentMultilineComment = "";
```

Comment:
What does setting it to `""` achieve here? Prevents the following APis getting the same comment? Is `currentMultilineComment` used anywhere else?